src/server.js

```diff
@@ -44,14 +44,16 @@ if (env.isDevelopment()) {
 const router = new express.Router()
 app.use('/v1', router)
 
-new TranslationRouter(router).mount()
-new PublicationRouter(router).mount()
-new MortgageRouter(router).mount()
+new AssetRouter(router).mount()
+new ParcelRouter(router).mount()
```

Comment:
`ParcelRouter` has to be the last one to be mounted... otherwise the `parcels/:x/:y` endpoint handles the requests for `/parcels/:address/mortgages`.

ie:

Request:

```
GET /parcels/0x0de1ec708665b93478bd0ed264e60b68a8663bb4/mortgages
```

Response:

```json
{
    "ok": false,
    "data": {},
    "error": "Invalid coordinate \"y\" must be an integer"
}
```

Although i think we should change the `/parcels/:address/mortgages` to be `/addresses/:address/mortgages` since the uri param is an identifier for the `addresses` resource, not `parcels`.

Reply:
I'll do both!